priiloader/source/executables.h

```diff
@@ -0,0 +1,206 @@
+/*
```

Comment:
why is this file needed? can't this be retrieved from the loader's source?

Reply:
I can replace it with a `#include "../../loader/include/executables.h"`, or it could be moved to `Shared`.

Reply:
i'd make the include just "executables.h", as the makefile includes the loader's include.
it would be way better in shared, but i'd like to keep the loader stand alone as i often use it in side projects to load all kind of stuff haha